acceso/templates/acceso/filtrar_imagenes.html

```diff
@@ -180,10 +186,9 @@ <h2><a href="#">{% trans "Photos" %} </a></h2>
 
 <div class="container">
   <!-- Images do not show up with the following line -->
-  <!-- <div id="grid" class="row my-shuffle-container"> -->
-  <div class="picture-item my-shuffle">
+  <div id="grid" class="row my-shuffle-container">
    {% for photo in photo_list %}
-   <figure class="picture-item" data-groups='[""]'>
+   <figure class="picture-item" data-groups=["apple"]>

```

Comment:
We'll want to change "apple" to something like {{ photo.filters }} so that the filters are added dynamically